starknet_py/contract_abi_resolver.py

```diff
@@ -0,0 +1,131 @@
+# Needed because of string typed Contract
```

Comment:
I think we can get rid of this. I suggest not accepting Contract at all & just use a client. For Argent we can perform a call using just a simple call. 
I don't want the low level interface to depend on high level interface, coupling them bidirectionally.